package.json

```diff
@@ -19,22 +19,29 @@
         "quick-lru": "^5.1.1"
     },
     "devDependencies": {
-        "@apify/eslint-config": "^0.1.3",
+        "@apify/eslint-config-ts": "^0.1.2",
+        "@types/node": "^16.4.13",
+        "@typescript-eslint/eslint-plugin": "^4.29.0",
+        "@typescript-eslint/parser": "^4.29.0",
         "body-parser": "^1.19.0",
+        "del-cli": "^4.0.1",
```

Comment:
So, why are we using this instead of `rimraf`? Other repos use `rimraf` and it works fine 👀 
Plus, while I'm not one to care about dependency trees/sizes, `del-cli` has a tree that's almost 10x the one of `del-cli` 😨

Reply:
> So, why are we using this instead of rimraf?

Dunno. It's just what I use. Will change this.